linkis-engineconn-plugins/spark/pom.xml

```diff
@@ -418,6 +418,12 @@
       <artifactId>kubernetes-model-core</artifactId>
       <version>${kubernetes-client.version}</version>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.doris</groupId>
+      <artifactId>spark-doris-connector-3.2_2.12</artifactId>
```

Comment:
Need to be compatible with multiple versions of spark

Reply:
Ok, thanks for reviewing the code